frameworks/Java/armeria/src/main/java/hello/App.java

```diff
@@ -0,0 +1,25 @@
+package hello;
+
+import hello.services.HelloService;
+import hello.services.PostgresDbService;
+import hello.services.PostgresFortunesService;
+
+import com.linecorp.armeria.common.SessionProtocol;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServerBuilder;
+
+public final class App
+{
+  public static void main(String[] args)
+  {
+    ServerBuilder sb = new ServerBuilder();
+
+    sb.port(8080, SessionProtocol.HTTP).annotatedService("/",
+        new HelloService()).annotatedService("/",
```

Comment:
(Usually we don't comment on highly-subjective matters of code style like this in TFB.  The following is more of a TechEmpower-internal comment than a TFB comment.)

When doing chained method calls like this, prefer to insert line breaks before each `.` rather than breaking between method parameters.  So in this case, something more like this:

```java
sb.port(8080, SessionProtocol.HTTP)
    .annotatedService("/", new HelloService())
    .annotatedService("/", new PostgresDbService())
    ...
```

Here's one explanation from Google's Java style guide, which pretty closely matches TechEmpower's preferred Java style. 
 See "4.5.1 Where to break":  https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break
> The prime directive of line-wrapping is: prefer to break at a **higher syntactic level**.

In this case, the `.annotatedService(...)` method calls occur at a higher syntactic level than the lists of input parameters.

I also really like the "rectangle rule" explanation from an older version of Google's JavaScript style guide.  They removed this section from more recent versions of the guide, but I still think it's good advice.  https://github.com/google/styleguide/blob/d012c726248bee61bdc20bd68a40fd67778739ae/jsguide.html#L650-L684
> When a source file is formatted, each subtree gets its own bounding rectangle, containing all of that subtree's text and none of any other subtree's.

In this case, consider the subtree `.annotatedService("/", new HelloService())`.  Under your current formatting, it's not possible to draw a rectangle around `.annotatedService("/", new HelloService())` that excludes all other code.